vpr/src/place/place.cpp

```diff
@@ -1396,6 +1410,10 @@ static e_move_result try_swap(float t,
 
     MoveOutcomeStats move_outcome_stats;
 
+    bool manual_move = get_manual_move_flag(); //whether the manual move info has been enabled or not
+    e_move_result manual_move_outcome;         //move_outcome from the user
```

Comment:
Move these declarations into the function I'm proposing below (better to declare variables right where they are used instead of ahead of time, plus that moves these into the proposed subroutine for readability).